ubuntu/14.04/standard/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM microsoft/vsts-agent:ubuntu-14.04
 
 # Install basic command-line utilities
 RUN apt-get update \
- && apt-get install -y --no-install-recommends \
+    && apt-get install -y --no-install-recommends \
```

Comment:
whoops .. reverting

Reply:
actually this follows the pep8 standard ...would you rather have these changes or revert ?

Reply:
I would rather revert it. This isnt a python project so we have no reason to follow pep8